io-engine/src/lvs/lvs_lvol.rs

```diff
@@ -754,12 +774,65 @@ impl LvsLvol for Lvol {
 
         info!("{:?}: creating snapshot '{}'", self, snapshot_name);
     }
+
+    /// Blobstore Common Callback function.
+    extern "C" fn blob_op_complete_cb(
+        arg: *mut c_void,
+        blob: *mut spdk_blob,
+        errno: i32,
+    ) {
+        let s = unsafe {
+            Box::from_raw(arg as *mut oneshot::Sender<(*mut spdk_blob, i32)>)
+        };
+        if errno != 0 {
+            error!("Blobstore Operation failed, errno {}", errno);
```

Comment:
would be good to include something about the blobstore, otherwise we don't know from the logs which blosbstor operation failed?

Reply:
Yes.. I think it make sense. I will update the blobstor information as part of error message in the next PR.